data/workspaces/boron.inv

```diff
@@ -264,6 +268,11 @@
                         <increment content="1" />
                         <value content="0" />
                     </Property>
+                    <Property type="org.inviwo.DirectoryProperty" identifier="layerDir">
```

Comment:
this should probably be considered a default state https://github.com/inviwo/inviwo/blob/96f60e567482831f60dd481ffc294f892c01e840/src/core/properties/fileproperty.cpp#L381 :/

Reply:
this is my fault I think...